### PR TITLE
[TestConsistency]bump tol of norm_fp16

### PR DESCRIPTION
**Pull request by shuhand0**
Bump error tolerance of  test_output_match_norm_cpu_float16  after cross-checking with cuda. MPS results are with default error tol with cuda.

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -1099,6 +1099,52 @@ def test_norm(self):
...
         self.assertEqual(res, res_cpu)
 
+        e = torch.tensor([[8.9453,  4.0859,  0.1230,  2.1367, -5.0000],
+                  [ 7.2773, -4.6953, -3.5586,  8.2812, -0.8789],
+                  [ 0.7119, -1.4854,  6.8633, -7.9805, -3.6562], 
+                  [-1.0195, -7.2695, -0.0264, -3.5078, -0.2900],
+                  [ 8.7656,  5.8984, -2.3125, -0.0352,  5.2812]],
+                  device='mps', dtype=torch.float16)
+        res = torch.norm(e, float(0.5))
+        # cuda result
```
Comment (ssaladis) [inline test/test_mps.py:1109]: Ziyu is using a separate .yaml file for coda results. Please see if it makes sense to move this data in that file.

Comment (kulinseth) [reply test/test_mps.py:1109]: Agreed, @shuhand0 can you move this to the yaml file.

**Review by kulinseth — APPROVED** [review]

File: test/test_mps.py [hunk 2]
```diff
@@ -8364,6 +8375,33 @@ def test_serialization_map_location(self):
...
+        self.params_ = params
+        self.out_ = out
+    def sample(self):
+        return self.args_ + self.params_
+    def expected(self):
+        return tuple(self.out_)
+
+CUDA_RESULT = dict()
+OP_UNIT_TEST = dict()
+with open("./test/cuda_results.yaml") as f:
```
Comment (ssaladis) [inline test/test_mps.py:8392]: Denis fixed this path and is merged to master. This path seems to create some issues

**Comment by kulinseth** [comment]
@shuhand0 , can you rebase the changes.

**Review by DenisVieriu97 — APPROVED** [review]
Looks good - I think one issue that might come from this approach (comparing CUDA with MPS) is when the test data changes and we'd have also to regenerate the data in the cuda file. There have been some cases where the samples used in the tests have changed between different pytorch versions.

**Pull request merged**